site/assets/scss/_styles_project.scss

```diff
@@ -194,7 +194,6 @@ div.feature-state-notice {
       font-size: 0.95em;
       font-weight: bold;
       color: #000;
-      background-color: #daeaf9;
```

Comment:
Why removing this?

Reply:
It's not necessary. Nothing change without it. 
But when I change a background I also need to change this color too.

Reply:
but if you remove this line will the background color be as before for non-deprecated features?

Reply:
Yes, It's just background for text. I think we don't need it.

<img width="763" alt="image" src="https://github.com/user-attachments/assets/b29eb056-aa3a-4e97-8757-59cf68140daa">

Reply:
Here's an example with `alpha` without that extra background:

<img width="763" alt="image" src="https://github.com/user-attachments/assets/81c2d9e8-ae49-4c07-8e21-47adc8cf77e4">

Nothing change.

Reply:
Ok, in the js core is the fallback so that it is not changed?

Reply:
No, we have a parent block background:

https://github.com/kubernetes-sigs/kueue/blob/bf6ea0d40f729490b9da1acd902c2af68f40f3f9/site/assets/scss/_styles_project.scss#L181

Reply:
Without this, the extra text has a transparent background and inherits the background color from its parent.

Reply:
hm, but if we keep this line it displays the same? In that case I wouldn't change it in this PR.

Reply:
We still need to change it with deprecation color. Because I have this problem

![image](https://github.com/user-attachments/assets/eee6fdf9-4ec4-4e2a-b0e6-a7c516b5fc3c)


And on this way we need to add one more line to replace it. For my opinion it's better to remove this line.

Reply:
I see, I also checked there is no regression in the display.